### ISSUE TITLE
Support ARIA

### ISSUE THREAD

**Issue by emchateau** [ouvroir/reframe#4]
Various web components libraries are centred around accessibility. Cf. React aria, etc. Accessibility should be a high priority for the project. Instead of seeing it as complexity, that should be embrasse broadly as a way to enhance functionality and linked to the responsive aspect of the components.
